easybuild/easyconfigs/r/ReFrame/ReFrame-3.9.2.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'PythonBundle'
+
+name = 'ReFrame'
+version = '3.9.2'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+req_py_majver = 3
+req_py_minver = 6
+
+use_pip = True
+
+exts_list = [
+    ('pip', '21.3.1', {
+        'use_pip': False,
+        'checksums': ['fd11ba3d0fdb4c07fbc5ecbba0b1b719809420f25038f8ee3cd913d3faa3033a'],
+    }),
+    ('wheel', '0.37.0', {
+        'checksums': ['e2ef7239991699e3355d54f8e968a21bb940a1dbf34a4d226741e64462516fad'],
+    }),
+    ('setuptools', '59.4.0', {
+        'source_tmpl': 'setuptools-%(version)s.tar.gz',
+        'checksums': ['b4c634615a0cf5b02cf83c7bedffc8da0ca439f00e79452699454da6fbd4153d'],
+    }),
+    ('reframe', version, {
+        'preinstallopts': "export PATH=%(installdir)s/bin:$PATH && "
```

Comment:
@boegel wouldn't it be better to keep the comment that was in `modextrapaths`, explaining why this is necessary?

Reply:
fixed!